spec/schedulers/TestScheduler-spec.ts

```diff
@@ -393,8 +393,9 @@ describe('TestScheduler', () => {
         const expected = ' -abcdef|';
 
         expectObservable(output).toBe(expected);
-        expectSubscriptions(obs1.subscriptions).toBe('^-----!');
-        expectSubscriptions(obs2.subscriptions).toBe('^------!');
+        expectObservable(output).toEqual(cold(expected));
+        expectSubscriptions(obs1.subscriptions).toBe(['^-----!', '^-----!']);
+        expectSubscriptions(obs2.subscriptions).toBe(['^------!', '^------!']);
```

Comment:
issue: It's not at all clear - to me, at least - why `obs1` and `obs2` are subscribed to twice. I think these expectations warrant an explanation. And if this is somehow related to `toEqual`, I don't think that helper should be made public.

Reply:
They're merged. And the merged observable is subscribed to on line 395 and again on line 396.

```ts
expectObservable(output).toBe(expected);
expectObservable(output).toEqual(cold(expected));
```

Reply:
I can add a comment I guess.